codechecker_common/hash.py

```diff
@@ -0,0 +1,12 @@
+
+import hashlib
+
+hashlib_md5_kwargs={'usedforsecurity':False}
```

Comment:
I'm not too familiar with this `usedforsecurity` flag. Do I understand correctly, that it changes the hash algorithm, but the resulting hash value will always be the same?
Could you add some comments about the motivation of this flag?
Is this relevant only for md5 or some other hash algorithms too?
This might be a theoretical question, but what happens if someone isn't aware of the existence of this utility function? Can we force its usage over `hashlib.md5`?